api/ask_astro/clients/weaviate_.py

```diff
@@ -3,19 +3,10 @@
 as well as text embeddings using the OpenAIEmbeddings from the LangChain library.
 """
 import weaviate
-from langchain.embeddings import OpenAIEmbeddings
 from langchain.vectorstores import Weaviate
 from weaviate import Client as WeaviateClient
 
-from ask_astro.config import AzureOpenAIParams, WeaviateConfig
-from ask_astro.settings import WEAVIATE_OPENAI_EMBEDDINGS_DEPLOYMENT_NAME, WEAVIATE_OPENAI_EMBEDDINGS_MODEL
-
-# Initialize OpenAI embeddings using the specified parameters.
-embeddings = OpenAIEmbeddings(
```

Comment:
why is this removed?

Reply:
This is dead code. Look around in the repo. `embeddings` is never used anywhere. This make sense because the embedding model should only ever be used during ingestion. And that is setup in the `schema.json` file. Not here